packages/luna/src/components/Chat/ChatMessage/ChatMessage.tsx

```diff
@@ -136,22 +150,28 @@ export const ChatMessage: React.FC<{
       <div
         className={cn(
           "tw-rounded-[15px] tw-relative tw-px-6 tw-py-3",
-          "tw-flex tw-items-center tw-gap-2 tw-justify-start",
-          pending && "tw-cursor-wait",
+          "tw-flex tw-items-end tw-gap-2 tw-justify-start",
+          pending && "tw-cursor-wait tw-opacity-50",
           {
-            "tw-bg-brand-100 dark:tw-bg-brand-100 tw-rounded-tl-none": !owner,
-            "tw-bg-brand-700 dark:tw-bg-brand-400 tw-rounded-tr-none": owner,
+            "tw-bg-natural-100 dark:tw-bg-brand-100 ": !owner,
+            "tw-bg-brand-100 dark:tw-bg-brand-400": owner,
             "tw-bg-destructive-700": error && !pending,
+          },
+          {
+            "tw-rounded-tl-none": !owner && firstMessage,
+            "tw-rounded-tr-none": owner && firstMessage,
           }
         )}
       >
-        <MessageCap
-          className={cn("-tw-top-1 tw-absolute", {
-            "-tw-right-0 [&>*]:tw-fill-brand-700": owner,
-            "-tw-right-0 [&>*]:tw-fill-destructive-700": error && !pending,
-            "-tw-left-0 tw-scale-x-[-1] [&>*]:tw-fill-brand-100": !owner,
-          })}
-        />
+        {firstMessage ? (
+          <MessageCap
+            className={cn("-tw-top-1 tw-absolute", {
```

Comment:
Let's add the dark mode styles in the Cap just like in the message box itself.
We may need to define/extend `tw-fill-brand-400` in tailwind first.